File: 012.jarakirefactor/djeiva/Sequencia8.java
package sequencia;

import java.util.Scanner;

public class Sequencia8 {

	/**
	 * @param args
	 */
	public static void main(String[] args) {		
		float custoHora, salario;
		int qtdHora;
		
		Scanner input = new Scanner(System.in);
		
		System.out.println("Informe quanto voce ganha por hora");		
		custoHora = input.nextFloat();
		
		System.out.println("Informe quantas horas voce trabalhou nesse mês");		
		qtdHora = input.nextInt();
		
		salario = qtdHora*custoHora;
		System.out.println("O salario correspondente as horas trabalhadas é: " + salario);	

	}
}

File: 012.jarakirefactor/test/ebin/sequencia8.erl
-module(sequencia8).

-compile(export_all).

-import(loop, [for/3, while/2]).

main(V_args) ->
    st:new(),
    st:get_new_stack(main),
    st:put_value({main, "args"}, {'String', V_args}),
    io:format("~s~n",
	      ["Informe quanto voce ganha por hora"]),
    st:put_value({main, "custoHora"},
		 {float, io:fread(">", "~f")}),
    io:format("~s~n",
	      ["Informe quantas horas voce trabalhou "
	       "nesse mês"]),
    st:put_value({main, "qtdHora"},
		 {int, io:fread(">", "~d")}),
    st:put_value({main, "salario"},
		 {float,
		  st:get_value(main, "qtdHora") *
		    st:get_value(main, "custoHora")}),
    io:format("~s~f~n",
	      ["O salario correspondente as horas trabalhadas "
	       "é: ",
	       st:get_value(main, "salario")]),
    st:get_old_stack(main),
    st:destroy().

